addons/web_settings_dashboard/static/tests/dashboard_tests.js

```diff
@@ -191,6 +195,136 @@ QUnit.module('settings_dashboard', function () {
+                    return $.when();
+                }
+                return this._super.apply(this, arguments);
+            },
+        });
+
+        var $loadTranslation = dashboard.$('.o_load_translations');
+
+        assert.strictEqual($loadTranslation.length, 1,
+            "should have button to load translations");
+
+        // Prevent the browser default behaviour when clicking on anything.
+        // This includes clicking on a `<a>` with `href`, so that it does not
+        // change the URL in the address bar.
+        $(document.body).on('click.o_test', function (ev) {
+            assert.ok(ev.isDefaultPrevented(),
+                "should have prevented browser default behaviour");
+            assert.strictEqual(ev.target, $loadTranslation[0],
+                "should have clicked on 'load a translation' button");
+            ev.preventDefault();
+        });
+
+        $loadTranslation.click();
+
+        $(document.body).off('click.o_test');
+
+        dashboard.destroy();
+    });
+
+    QUnit.test('Prevent default behaviour when clicking on set up company', function (assert) {
+        assert.expect(3);
+
+        var dashboard = createDashboard({
+            dashboards: ['company'],
+            mockRPC: function (route, args) {
+                if (route === '/web_settings_dashboard/data') {
+                    return $.when({
+                        company: {
+                            company_name: 'MyCompany'
+                        }
+                    });
+                }
+                return this._super.apply(this, arguments);
+            },
+        });
+
+        var $setupCompany = dashboard.$('.o_setup_company');
+
+        assert.strictEqual($setupCompany.length, 1,
+            "should have button to set up company");
+
+        // Prevent the browser default behaviour when clicking on anything.
+        // This includes clicking on a `<a>` with `href`, so that it does not
+        // change the URL in the address bar.
+        $(document.body).on('click.o_test', function (ev) {
+            assert.ok(ev.isDefaultPrevented(),
+                "should have prevented browser default behaviour");
+            assert.strictEqual(ev.target, $setupCompany[0],
+                "should have clicked on 'setup company' button");
+            ev.preventDefault();
+        });
+
+        $setupCompany.click();
+
+        $(document.body).off('click.o_test');
+
+        dashboard.destroy();
+    });
+
+    QUnit.test('Prevent default behaviour when clicking on browse apps', function (assert) {
+        var done = assert.async();
+        assert.expect(3);
+
+        // Determine when apps have been loaded
+        var def = $.Deferred();
+        var originalLoadApps = Dashboard.prototype.load_apps;
+        testUtils.patch(Dashboard, {
+            load_apps: function () {
+                return originalLoadApps.apply(this, arguments).then(function (res) {
+                    def.resolve();
```

Comment:
is this really async? Tests should not do any RPC...

Reply:
The include here is problematic: https://github.com/odoo/odoo/blob/12.0/addons/iap/static/src/js/dashboard.js#L15